### DISCUSSION TITLE
Expanded Looking For Filter Options

### DISCUSSION THREAD

**Discussion by ajbott** [PullDating/Org#210] [Ideas & Features]
### **1. Casual Connections**
- **One-Time Encounter**: For those interested in a single, no-strings-attached meeting.
- **Friends with Benefits**: A setup where friends engage in casual sex without a romantic relationship.
- **Casual Dating**: Non-exclusive, short-term dating with no long-term commitment expectations.
- **Activity Partner**: Seeking someone for shared activities like hiking, concerts, or sports events, without romantic expectations.

### **2. Committed Relationships**
- **Short-Term Relationship**: A relationship expected to last for a few months, focused on enjoying the present without long-term plans.
- **Long-Term Relationship**: Seeking a deep, committed relationship with long-term potential.
- **Life Partner**: Looking for a lifelong commitment, potentially leading to marriage or a partnership equivalent.
- **Transitioning Relationship**: Open to starting casually with the potential for evolving into a more serious, long-term relationship.

### **3. Specific Arrangements**
- **Travel Companion**: Seeking a partner for traveling, with or without romantic implications.
- **Digital Connection**: Interested in an online-only relationship, ranging from platonic friendships to virtual romances.
- **Polyamorous Relationships**: Open to multi-person relationships, with clear communication and consent from all parties involved.
- **Open Relationships**: In a relationship but open to additional partners, with the consent of everyone involved.

### **4. Friendship & Networking**
- **Platonic Friendship**: Looking to expand social circles without romantic or sexual intentions.
- **Professional Networking**: Seeking connections for professional development and networking, not for dating.

### **5. Experimenting**
- **Exploratory Dating**: Open to trying different types of relationships to understand personal preferences better.
- **Short-Term to Long-Term**: Initially interested in short-term dating but open to the possibility of it becoming long-term.

### **6. Unique Situations**
- **Co-Parenting Partnership**: Looking for a platonic partnership focused on raising children together.
- **Spiritual/Intellectual Connection**: Seeking a relationship centered around spiritual growth, intellectual exchange, or creative collaboration.

### **7. Specific Interests or Lifestyles**
- **Health & Wellness Companion**: Interested in a partner to share in health, fitness, or wellness activities.
- **Creative Collaborator**: Seeking a connection based on shared creative pursuits, such as art, music, or writing.

**Note**: Users can select multiple options and are encouraged to communicate openly about their intentions and expectations to ensure clarity and mutual understanding.

**Comment by Oshyan** [comment]
I like the idea of allowing users to comprehensively and clearly indicate their preferences and intentions. But am also wary of overloading people with options, especially if many won't get used. User interface will be highly important here. It's probably worth looking at an app like Feeld or maybe OKC for how they implement similar things (both have a lot of options for what the user is looking for). 

Also not sure "Networking" should necessarily be covered in the same app or section, honestly. It's a nice idea, and "Friendship" is reasonable I think. But I don't think mingling the idea of business connections here is a good thing to do.
